siiptool/scripts/subregion_capsule.py

```diff
@@ -169,6 +160,10 @@ def convert_arg_line_to_args(arg_line):
         print('All-or-none of the certificate files must be provided.')
         exit(2)
 
+    
+    # Check if openssl is installed or at given path
+    utils.check_for_tool('openssl', 'version', tool_path=args.SigningToolPath)
```

Comment:
Mohamed - did you test without --signing-tool-path argument from commend line. Does it behave correctly?

Reply:
Yes